R/exportDataQuality.R

```diff
@@ -0,0 +1,92 @@
+
+exportDataQuality <- function(rcon, prefix,
+                                     ...){
+  UseMethod("exportDataQuality")
+}
+
+exportDataQuality.redcapApiConnection <- function(rcon, prefix,
+                                                       ..., 
+                                                       error_handling = getOption("redcap_error_handling"), 
+                                                       config = list(), 
+                                                       api_param = list()){
+  ###################################################################
+  # Argument Validation                                          ####
+  
+  coll <- checkmate::makeAssertCollection()
+  
+  checkmate::assert_class(x = rcon, 
+                          classes = "redcapApiConnection", 
+                          add = coll)
+  
+  checkmate::assert_class(x = prefix, 
+                          classes = "character", 
+                          add = coll)
+  
+  error_handling <- checkmate::matchArg(x = error_handling, 
+                                        choices = c("null", "error"),
+                                        .var.name = "error_handling",
+                                        add = coll)
+  
+  checkmate::assert_list(x = config, 
+                         names = "named", 
+                         add = coll)
+  
+  checkmate::assert_list(x = api_param, 
+                         names = "named", 
+                         add = coll)
+  
+  checkmate::reportAssertions(coll)
+  
+  ###################################################################
+  # Build the query list                                         ####
+  
+  url <- paste0("https://redcap.vanderbilt.edu/api/?prefix=", prefix, "&page=export&type=module&NOAUTH&pid=", rcon$projectInformation()$project_id)
```

Comment:
The url should come from the object. I would search the code for examples of building the post string.

Reply:
updated to build url from connection object.